byoqm/dashboard/dashboard.py

```diff
@@ -37,6 +38,27 @@ def _check_date(self, date, start_date, end_date):
             return False
         return True
 
+    def _get_files(self):
+        figure_files = os.listdir("./figures")
+        figure_files.remove("__pycache__")
```

Comment:
I'd be careful to delete something that python (I think? Something python/poetry related) auto-generates. Maybe we could just check if the file is a directory or a file? I know you can do this with the `Path` object in `pathlib` pretty easily.

Reply:
I researched what the __pycache__ folder is, and it seems that it contains byte-code compilations of our python files, which means executing our program will be just a little bit slower if we delete it repeatedly.